src/components/Blog/Post/index.tsx

```diff
@@ -111,7 +111,7 @@ const BlogPost: React.SFC<IBlogPostData> = ({
               ))}
             </div>
           )}
-          {commentsUrl && ready && !error && (
+          {commentsUrl && ready && !error && typeof result === 'number' && (
```

Comment:
just curious, should we raise na exception/log if it's not a number?

Reply:
Changed it to simpler check, because I added return types to all utils. I'm not sure why we should want log it. Such checks necessary for TS typings system. But if you want to be sure that API returns correct data - I think we should validate it in api helpers, not in UI components.